### PR TITLE
Feature/populate potential inc database

### PR DESCRIPTION
**Pull request by nicolascarva**
Naming: feature/descriptive-name

Reviewers: minimum 2 (Chris and/or 2 from Web or DS respectively and not person who submitted PR)

Description: 
  What is the motivation for changes?
  What specific changes were made?

Submitter Checklist: 
  - [x] Small scope (1-2 components)
  - [x] Not too many concerns in a single commit
  - [x] Remove extraneous comments (code in comments), print statements, to-dos, console.logs, extra fluff
  - [x] Atomic, descriptive commit messages
  - [x] Consistent formatting

Reviewer Checklist:
  - [x] Small scope (1-2 components)
  - [x] Comments, print statements, to-dos, console.logs, extra fluff were removed
  - [x] Give feedback no matter what conversations happened elsewhere, document here
  - [x] Comment on the line of code itself (blue plus box) in files changed tab

### CODE REVIEW

File: app/create_db_tables.py [hunk 1]
```diff
@@ -1,21 +1,27 @@
...
 import psycopg2
 from dotenv import load_dotenv
 load_dotenv()
 
+
+
 def initialize_police_table():
 
 
+
```
Comment (jalirkan) [inline app/create_db_tables.py:16]: This was a much needed fix to the schema

**Comment by jalirkan** [comment]
Good fix

**Review by jalirkan — APPROVED** [review]
Good fix

**Comment by israelaikulola** [comment]
Awesome

**Review by njfleck24 — APPROVED** [review]
The regex may confuse future users but it all looks good

**Pull request merged**